### DISCUSSION TITLE
Help to recover old account

### DISCUSSION THREAD

**Discussion by AlamBaierVale** [community/community#56399] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The email associated with my github account no longer exists. It was replaced by another one in my organization. As I don't remember the password, I am unable to access the account and I have proven repositories that I would like to regain access to.
Is there any way to recover?
Today we still have 2 staff members who are able to see the repository but only with basic access as I was the sole owner.

**Comment by airtower-luna** [comment]
*If* anyone can help you it's [GitHub support](https://support.github.com/request), but with neither mail address nor password that seems doubtful to me.

Maybe your organization can (at least temporarily) restore that mail address? Then you could just request a [password reset](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/updating-your-github-access-credentials#requesting-a-new-password).

**Reply by AlamBaierVale** [reply]
Unfortunately I left the company for 2 months and now that I returned, I was provided with a new login with a new email... the accounts are Microsoft AD, so I think IT could even restore it, but that would be extremely bureaucratic here.
I'm thinking of asking the other team members to fork the repository, could this cause any future issues? Because it is somehow tied to the original repository right?

**Reply by airtower-luna** [reply]
> I'm thinking of asking the other team members to fork the repository, could this cause any future issues?

Depends on what you mean by issues. :sweat_smile: 

> Because it is somehow tied to the original repository right?

Yeah, though you can have it detached. If you want to work with a new repository it's probably the easiest to just create a new repository and push the code there. The main catch is that issue/PR numbers will start fresh, too (but that'd be the same with a fork), which may cause trouble if you used them in commit messages.
